### PR TITLE
The structure whose icons are used as images in all flutters has been converted to Composer Icon format.

### PR DESCRIPTION
**Pull request by yapmakutu**

### CODE REVIEW

**Review by Ayazelar — COMMENTED** [review]
Image "alt" props shouldn't be an empty string, it can be something like the title or some other text. All HTML code should be SEO optimized.

File: editor-components/footer/footer1/footer1.tsx [hunk 1]
```diff
@@ -558,8 +561,12 @@ class Footer1Page extends BaseFooter {
               )
             )}
             <div className={this.decorateCSS("subscribe")}>
-              <h1 className={this.decorateCSS("subscriptionTitle")}>{this.getPropValue("subscriptionTitle")}</h1>
-              <p className={this.decorateCSS("subscription-description")}>{this.getPropValue("subscriptionDescription")}</p>
+              <h1 className={this.decorateCSS("subscriptionTitle")}>
```
Comment (dogusmiuw) [inline editor-components/footer/footer1/footer1.tsx:564]: if prop is empty, then remove relevany elements. do this for all props

File: editor-components/footer/footer10/footer10.tsx [hunk 2]
```diff
@@ -13,10 +14,10 @@ class Footer10Page extends BaseFooter {
...
     });
     this.addProp({
-        type: "page",
-        key: "ref-page",
-        displayer: "Referral Url",
-        value: "https://www.google.com/"
+      type: "page",
+      key: "ref-page",
+      displayer: "Referral Url",
+      value: "https://www.google.com/",
```
Comment (dogusmiuw) [inline editor-components/footer/footer10/footer10.tsx:20]: use empty string for link props

File: editor-components/footer/footer10/footer10.tsx [hunk 3]
```diff
@@ -13,10 +14,10 @@ class Footer10Page extends BaseFooter {
       value: "© 2022 Dental Excellence Turkey | ",
     });
     this.addProp({
-        type: "page",
-        key: "ref-page",
-        displayer: "Referral Url",
-        value: "https://www.google.com/"
+      type: "page",
+      key: "ref-page",
```
Comment (dogusmiuw) [inline editor-components/footer/footer10/footer10.tsx:18]: use camelCase for keys

File: editor-components/footer/footer10/footer10.tsx [hunk 4]
```diff
@@ -130,27 +128,33 @@ class Footer10Page extends BaseFooter {
...
+    };
     return (
       <div className={this.decorateCSS("container")} style={styling}>
         <div className={this.decorateCSS("max-content")}>
           <div className={this.decorateCSS("content")}>
             <div className={this.decorateCSS("top-content")}>
               <span className={this.decorateCSS("top-text")}>
                 {this.getPropValue("top-text")}
-                <a href={this.getPropValue("ref-page")}>{this.getPropValue("ref-text")}</a>
+                <a href={this.getPropValue("ref-page")}>
```
Comment (dogusmiuw) [inline editor-components/footer/footer10/footer10.tsx:140]: if prop is empty, do not render relevant elements

**Review by dogusmiuw — CHANGES_REQUESTED** [review]
apply the change requests for all the files

**Pull request closed without merging**